test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -296,7 +300,10 @@ public void resume(int n) {
   }
 
   public void start(int n) {
-    execute("node" + n, "start");
+    List<String> cmdAndArgs = Lists.newArrayList("node" + n, "start");
+    overrideJvmVersionForDseWorkloads()
+        .ifPresent(jvmVersion -> cmdAndArgs.add(String.format("--jvm_version=%d", jvmVersion)));
+    execute(cmdAndArgs.toArray(new String[0]));
```

Comment:
I don't _love_ the duplication of the --jvm_version handling here across two instances of start() but fixing it is harder than one might like.  There's a dependency not only on class member fields (in the form of jvmArgs) but also method call args (in the form of  n in the node case) that make abstracting it kinda ugly.  It's prolly okay to go with for now; we can always improve it later if somebody hits upon a good idea for doing so.

Reply:
Agreed. This is just test code so we can tidy it up later if needed.